app/src/main/java/com/summer/practice/tvtracker/LoginActivity.kt

```diff
@@ -0,0 +1,37 @@
+package com.summer.practice.tvtracker
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+
+class LoginActivity : AppCompatActivity() {
```

Comment:
let's add this class in another package called login, in the future we will have more classes for this feature